### PR TITLE
Review notes from #13194

### PR DESCRIPTION
**Pull request by xster**

### CODE REVIEW

File: packages/flutter/lib/src/cupertino/page_scaffold.dart [hunk 1]
```diff
@@ -39,7 +39,8 @@ class CupertinoPageScaffold extends StatelessWidget {
   /// Widget to show in the main content area.
   ///
   /// Content can slide under the [navigationBar] when they're translucent with
-  /// a [MediaQuery] padding hinting the top obstructed area.
+  /// a [MediaQuery] padding signaling the top obstructed area via
+  /// [MediaQueryData.padding].
```
Comment (Hixie) [inline packages/flutter/lib/src/cupertino/page_scaffold.dart]: I'm not sure I really understand what this paragraph is trying to say.

File: packages/flutter/lib/src/cupertino/page_scaffold.dart [hunk 2]
```diff
@@ -50,8 +51,9 @@ class CupertinoPageScaffold extends StatelessWidget {
     if (navigationBar != null) {
       final MediaQueryData existingMediaQuery = MediaQuery.of(context);
 
-      // TODO(https://github.com/flutter/flutter/issues/12912):
+      // TODO(xster):
```
Comment (Hixie) [inline packages/flutter/lib/src/cupertino/page_scaffold.dart]: nit: you can remove the newline after the colon, too. that way grepping for "TODO" will show the message

**Comment by Hixie** [comment]
LGTM

**Comment by xster** [comment]
#13194 is reverted. I'm gonna close this and do this in the revert of the revert.

File: packages/flutter/lib/src/cupertino/page_scaffold.dart (hunk 1 above)
Comment (xster) [reply packages/flutter/lib/src/cupertino/page_scaffold.dart]: reworded again in #13440

**Pull request closed without merging**